docs/en/drivers/ios-xcuitest.md

```diff
@@ -125,15 +125,9 @@ following desired capabilities:
 
 ### Optional Setup
 
-* Install FBSimulatorControl for better handling of various iOS Simulator operations,
+* Install idb for better handling of various iOS Simulator operations,
```

Comment:
It probably makes sense to mention that idb is only supported since 1.13.1

Reply:
1.13.1 has not been released yet. 1.14.0-beta was released.
I would like to update either 1.13.1 or 1.14.0 here after the version has been fixed